config.py

```diff
@@ -155,18 +155,42 @@
 set_channel_types = None
 
 ###############################################################################
-# MAXFILTER PARAMETERS
-# --------------------
+# MAXFILTER AND MAXWELL FILTER PARAMETERS
```

Comment:
what's the difference? MAXFILTER is more an abbrev of MAXWELL FILTER

Reply:
To me, MaxFilter to me is the Elekta thing, Maxwell filter is a more generic term. Is that not the case?

Reply:
@SophieHerbst I would be interested in your opinion here too :)

Reply:
but we don't have maxfilter support here. All we do is what mne calls maxwell_filter

Reply:
We do this `find_bad_channels_maxwell` which is supposed to be _kind of_ like `MaxFilter`, but yes maybe I should simply change this to read "MAXWELL FILTER PARAMETERS"

Reply:
solved in 2c06272